test/dummy/Gemfile

```diff
@@ -4,3 +4,4 @@ gem 'rails', '4.2.0'
 
 gem 'rails-api'
 gem 'apipie-rails', '>= 0.3.1'
+gem 'rdoc'
```

Comment:
Why do you need rdoc for this patch?

Reply:
You are right. I had to add rdoc to build the dummy app cache for the tests. When re-checked in clean setup it was not needed anymore. Removed.